### DISCUSSION TITLE
Plug-in and connector

### DISCUSSION THREAD

**Discussion by AnthoM33** [vendurehq/vendure#1796] [General]
Hello,

- Interested in the search plug-in I saw on this [post](https://github.com/vendure-ecommerce/vendure/discussions/1668), would you have an idea of the price you will put in place ?
- Can we integrate our own plug-in without blocking ?
- Do you have any connectors built on ERP (Odoo, ....) and PIM ?

Thank you.

**Comment by michaelbromley** [comment]
Hi!

1. We've not decided on any final pricing details (the marketplace stuff is still probably several months away from release), but in the ballpark of EUR500/year is a reasonable estimate.
2. Of course! Build whatever you need! 😄 
3. We're internally working on a PimCore connector that will likely be open-sourced at some point. No promises on any dates right now though.
